app/constants/server.js

```diff
@@ -3,6 +3,11 @@ const server = {
   hstsMaxAge: 63072000 // 2 yrs
 }
 
+const statusCodes = {
```

Comment:
I personally prefer different consts rather than an array of constants, or say statusCodes.code404
Just easier to use in the code.